### ISSUE TITLE
Show a shareable link when user not on network

### ISSUE THREAD

**Issue by seanrelay** [relaycc/receiver#117]
When a user types in an address or lens not on the network yet, they see this.
<img width="439" alt="Screen Shot 2022-11-11 at 6 52 59 PM" src="https://user-images.githubusercontent.com/102389629/201453261-c5fafacf-f5ea-41bb-8226-6faa767bdf23.png">

We need to add a sentence saying, "Help get them online by sharing relay.cc/u/[userAddress] to have them message you."

**Comment by killthebuddh4** [comment]
@seanrelay I'd like to do something like

Help them get web3 encrypted messaging by sharing your [Relay Profile](https://relay.cc/u/seanwbren.eth).

Specifically, I think the display text should be "Relay Profile" because it will avoid the long address and so the user won't have to parse the URL to know what they're linking to.

**Comment by seanrelay** [comment]
Sure, but I meant for it to be copy pastable super easy. Even easier is if we show that, and then have a copy icon that says "Link Copied" on success. On phones a copy icon is necessary even.

**Comment by BrianS111** [comment]
![DC434180-0E0F-4A8A-AF07-CB4E786876F5](https://user-images.githubusercontent.com/81312527/202085706-95e12d37-73d6-4194-b3c0-075b3301c9f1.jpeg)
Let me know what needs to be adjusted as far as looks are concerned

**Comment by seanrelay** [comment]
@BrianS111 looks good. 

Can you make Relay Profile colored blue and clickable as a link? 

When the copy button is clicked does it say "Link Copied" ?

**Comment by BrianS111** [comment]
https://user-images.githubusercontent.com/81312527/202220173-03241663-44c9-469e-80c1-c86cf1d56768.mov

How does this look?

**Comment by BrianS111** [comment]
@seanwbren When you say open a new tab, you want it to open a new tab to your own relay profile?

**Comment by killthebuddh4** [comment]
@BrianS111 

* The icon needs to be roughly half the current size, roughly the same size as the line height
* The icon should be black because it's not part of the link

**Comment by BrianS111** [comment]
> @BrianS111
> 
> * The icon needs to be roughly half the current size, roughly the same size as the line height
> * The icon should be black because it's not part of the link
> 
> For reference <img alt="image" width="402" src="https://user-images.githubusercontent.com/100799178/202245586-e69d6439-33e5-4d36-b5fb-b4e921c61307.png">
> 
> * PS, just thinking out loud: it might look nice to have the final "Relay Profile " line with slightly larger verical spacing so it's separated from the rest of the text.

Are you implying that clicking the copy icon and clicking "Relay Profile" do two different things? The Icon only copies the URL and Relay Profile only opens a new tab to their relay profile without copying the link?

**Comment by killthebuddh4** [comment]
> Are you implying that clicking the copy icon and clicking "Relay Profile" do two different things? The Icon only copies the URL and Relay Profile only opens a new tab to their relay profile without copying the link?

Yes, the "Relay Profile" text is a link that opens into a new tab. The icon copies the URL from the "Relay Profile" link. I'm pretty confident this is what @seanrelay has been talking about in his last few comments as well.
